roles/odf_setup/tasks/local-storage-operator.yml

```diff
@@ -73,5 +73,5 @@
             annotations:
               "{{ default_storageclass_annotation }}"
   when:
-    - not enable_ocs | default(false) | bool
+    - ocs_install_type == 'internal'
```

Comment:
you're already checking this in main.yml, do you need it here?

Reply:
yes, not needed. Thanks!